torchao/models/llama/tokenizer.py

```diff
@@ -0,0 +1,110 @@
+import os
```

Comment:
I guess these are taken from gpt-fast. Can we add a comment somewhere to mention that?